var/spack/repos/builtin/packages/py-pyscf/package.py

```diff
@@ -11,23 +11,28 @@ class PyPyscf(PythonPackage):
     """PySCF is a collection of electronic structure programs powered
     by Python."""
 
-    homepage = "https://sunqm.github.io/pyscf/"
+    homepage = "https://pyscf.org"
     git = "https://github.com/pyscf/pyscf"
```

Comment:
Would it be possible to download from PyPI instead? Then each release can be checksummed.

Reply:
Agreed that checksums would be preferred. Will investigate if the devs are consistent in updating PyPI and tackle next release if so.

Reply:
Updates